File: java/unit_test/BasicMathTest.java
import org.junit.Test;
import static org.junit.Assert.assertEquals;

public class BasicMathTest{
    BasicMath bm = new BasicMath();

    @Test
    public void testAddition(){
	assertEquals(3,bm.addition(1,2));
    }

    @Test
    public void testMultiply(){
	assertEquals(3,bm.multiply(1,3));
    }
}

File: go/unit_testing/basic_math_test.go
package unit_testing

import "testing"

func TestAddition(t *testing.T){
  sum := addition(1,2)
  if sum != 3 {
    t.Error("Valor esperado: 3, valor regresado: ", sum)
  }
}

func TestMultiply(t *testing.T){
  mult := multiply(1,2)
  if mult != 2 {
    t.Errorf("Valor esperado: 2, valor regresado: ", mult)
  }
}